models/istio_validation.go

```diff
@@ -329,6 +335,7 @@ func (summary *IstioValidationSummary) mergeSummaries(cs []*IstioCheck) {
 		} else if c.Severity == WarningSeverity {
 			summary.Warnings = summary.Warnings + 1
 		}
+		summary.ObjectCount = summary.ObjectCount + 1
```

Comment:
I see this is consistent with the above, and it's subjective.  But for me either var++ or  var += 1 is a more clear way to increment.  Certainly still fine as is.

Reply:
Yep, it is an old habit from ruby times. Ruby core decided not to add this feature. Don't remember the rationale behind. Changed :beer: